sdf.md

```diff
@@ -165,6 +161,20 @@ entity:
 
 [^status]
 
+SDF is designed to be an extensible format.
+The present document constitutes the base specification for SDF; we
+speak of "base SDF" for short.
+In addition, SDF extensions can be defined, some of which may make use
+of extension points specifically defined for this in base SDF.
+For other extensions, it may be necessary to indicate in the SDF
+document that a specific extension is in effect; see
+{{information-block}} for details of the `feature` quality that can be
+used for such indications.
+Base SDF does not define a "version" concept for the SDF specification
+itself (as opposed to SDF documents); however there may be occasional
```

Comment:
I guess we can either drop the "may" part or leave a note without "may" depending on if we end up using such notes of earlier versions. Might be better without them for the final RFC though since, while useful for spec development, might be confusing for a final spec.

Reply:
OK, I made new issue for the cleanup:

https://github.com/ietf-wg-asdf/SDF/issues/127

Let's decide this in the interim.